### ISSUE TITLE
Consider converting DFR to a class of AHU

### ISSUE THREAD

**Issue by tasodorff** [google/digitalbuildings#300]
The duct furnaces currently observed are essentially AHUs with heating-only (and in observed instances some type of recirc damper).

They handle outside air and return air, and meet the current definition of an AHU. Either propose an additional rule for distinguishing DFR from AHU, or consolidate.

**Comment by tasodorff** [comment]
@cstirdivant does this make sense to you? Argue with me.

**Comment by cstirdivant** [comment]
I agree we can consolidate, however I would not consider all DFRs part of the AHU class. Many units that are currently implementing DFR types do not handle the outside air themselves, but are instead recirculating return air (with bypass) or receiving air from a common header (supplied by DOAS).  For this reason, I suggest we reclassify into AHU, CU, and FCU classes as needed.
